tests/data_631g.gbs

```diff
@@ -951,4 +951,4 @@ D   3   1.00
       0.6251213000D+01       0.7120146000D+00
 D   1   1.00
       0.1979236000D+01       1.0000000
-****
+****
```

Comment:
It seems like this commit was introduced because of the editor stripping the newlines at the end of file.

Reply:
Memory does not serve me well on this one. I don't think we want to merge #110 though, I think that #111 is the better one, it may be worth keeping #110 floating around but the version introduced by #111 is a lot more memory friendly and I believe just simpler in general.